packages/vite/src/node/plugins/css.ts

```diff
@@ -1016,10 +1016,10 @@ const scss: SassStylePreprocessor = async (
     resolvers.sass(url, importer).then((resolved) => {
       if (resolved) {
         rebaseUrls(resolved, options.filename, options.alias)
-          .then(done)
-          .catch(done)
+          .then((data) => done?.(data))
+          .catch((data) => done?.(data))
```

Comment:
line 1019 and 1020 looks unnecessary. when I remove change, there's no type error message

Reply:
Theoretically yes, but the way I wrote it, it's much more safe.
If you hover over the type of `done`, you can see that it is now also `undefined`-able. Due to I'm not quite sure what would be happen when it is really `undefined`, it is now more clear that we are calling it only when it really exists, otherwise we break out with a `void`. You cannot read this when we just pass `done` as a reference.

Reply:
looks good to me.